resources/assets/helpers.js

```diff
@@ -22,11 +22,21 @@ export function calculateAge(date) {
 };
 
 export function getImageUrlFromProp(photoProp) {
-	const photo_url = photoProp['url'];
+	var photo_url;
 
-	if (photo_url == "default") {
-	  return "https://www.dosomething.org/sites/default/files/JenBugError.png";
-	}
+  // Sometimes we get the url right on the post and sometimes it is nested under
+  // media (in cases where it goes through the PostTransformer), so handle both cases
```

Comment:
Let's add a `@TODO` to normalize this so our data is always passed through `PostTransformer` before getting here.